src/pyrokinetics/gk_code/gs2.py

```diff
@@ -241,8 +234,14 @@ def get_local_species(self):
             for pyro_key, gs2_key in self.pyro_gs2_species.items():
                 species_data[pyro_key] = gs2_data[gs2_key]
 
-            species_data.vel = 0.0 * ureg.vref_most_probable
-            species_data.inverse_lv = 0.0 / ureg.lref_minor_radius
+            domega_drho = (
```

Comment:
I don't think we want to use uprim here. The previous form with `g_exb` looks closer to what we'd want?

Reply:
If a input file has `g_exb` then that gets read directly into Numerics` here

https://github.com/pyro-kinetics/pyrokinetics/blob/995094d82afeeb5ab9ed931713ac12b20deed903/src/pyrokinetics/gk_code/gs2.py#L426-L430

And then written to file here

https://github.com/pyro-kinetics/pyrokinetics/blob/995094d82afeeb5ab9ed931713ac12b20deed903/src/pyrokinetics/gk_code/gs2.py#L575

So this part of the code is sort of bypassed when setting ExB shear.

The reasoning here is that when loading a GS2 input file in, we need to map uprim to something (if it is set) and the most appropriate thing I could see was `domega_drho`.

Reply:
OK thanks, that makes sense. I'm not sure how widely used uprim is by those currently running GS2.

Reply:
Should I just remove it from GS2 entirely then?

Reply:
It might be worth doing so and then we can see if anyone raises it as an issue/request in the future.

Reply:
I'll have to set `domega_drho` to 0 so when we convert between codes we don't accidentally add in the PVG term

Reply:
Will `g_exb` still be set when doing the code comparison though?